### ISSUE TITLE
[Bug]: DUP Glitch

### ISSUE THREAD

**Issue by CorsaCapotado** [AllTheMods/ATM-10#829]
### Possible Fixes

Yes

### Modpack Version

1.2

### What happened?

i was duplicatin ores with the bronze hammer, and the system just double the amount by clicks, im using the integrated dynamics plus the sophisticated storage 
![aa2](https://github.com/user-attachments/assets/0b9fecb0-cd2c-460f-b160-b0526326babb)
![aaaaaaaa](https://github.com/user-attachments/assets/f103295f-f65f-4f65-ba3a-e77612cd44ff)
![aaaaaaaa](https://github.com/user-attachments/assets/a0c06d24-51d7-4737-8d2f-ade84a32494c)

**Comment by Curly0** [comment]
Yes, I had the same problem, I was also using a bronze hammer in the integrated dynamics terminal. It's like it glitches the ore in the terminal, it even allowed me to just pull infinite ore without even having a hammer in the terminal.
